eth_tester/backends/mock/factory.py

```diff
@@ -357,7 +357,7 @@ def make_block_from_parent(parent_block, overrides=None):
     if 'timestamp' in overrides:
         yield 'timestamp', overrides['timestamp']
     else:
-        yield 'timestamp', parent_block['timestamp'] + 15
```

Comment:
It seems like this `15` was arbitrary (probably just the avg block time) - as far as I can tell there are no other side affects to changing this.